modules/contacts/modules.php

```diff
@@ -176,7 +176,7 @@ protected function output() {
                     '<img alt="'.$this->trans('Send To').'" width="16" height="16" src="'.Hm_Image_Sources::$person.'" /></a> '.
                     '</d><td>'.$this->html_safe($contact->value('type')).'<td><span class="contact_src">'.
                     ($contact->value('source') == 'local' ? '' : $this->html_safe($contact->value('source'))).'</span>'.
-                    '</td><td>'.$this->html_safe($name).'</td>'.
+                    '</td><td>'.$this->html_safe_decoded($name).'</td>'.
```

Comment:
If we decode html entities, won't this bring security issues listing the names here in html context? I think it defeats the purpose of using html_safe in the first place.